javascript/javascript3/week3/shopping.js

```diff
@@ -0,0 +1,128 @@
+// sometimes it showing 403 failed loading error
+const user1 = fetch(
+  "https://api.github.com/search/repositories?q=user:benna100"
+)
+  .then((response) => response.json())
+  .then((bennaData) => {
+    return `FullName : ${bennaData.items[1].full_name}<br>
+    url : ${bennaData.items[1].owner.url}<br>
+    Owner : ${bennaData.items[1].owner.login}`;
+  });
+
```

Comment:
For the promise of the user data, you should stop at response.json().
Since you are doing .then and using the data, this is not a promise anymore, it returns the string that you form. 

Everything after is repetitive code that is done for every user, and it is also